scripts/goldpolish-target-extract-seq.py

```diff
@@ -0,0 +1,178 @@
+"""Extracts regions to polish with additional flanks into fasta file"""
+# imports
+import argparse
+import csv
+import re
+import btllib
+from collections import namedtuple
```

Comment:
```suggestion
from collections import namedtuple
import btllib
```
Normally standard imports are placed before third-party libraries